modules/integration_azure-cdn/conf/00-latency.yaml

```diff
@@ -0,0 +1,24 @@
+module: Azure CDN
+name: latency
+filtering: "filter('resource_type', 'microsoft.cdn/profiles') and filter('primary_aggregation_type', 'true')"
+id: cdn_total_latency
+
+transformation: true
+aggregation: true
+
+signals:
+  signal:
+    metric: "TotalLatency"
+    rollup: max
```

Comment:
This should be removed, It would make the metric less readable in detector and also trigger more alerts than needed.